mmv1/templates/terraform/examples/scc_v2_folder_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,12 @@
+resource "google_folder" "default" {
+	display_name = "A test folder mute config"
+	parent       = "folders/${var.org_id}"
+}
+
```

Comment:
You will need to add an iam_member resource granting the correct permissions for this folder.

Reply:
addressed